boards/amd/amd_acp_6_0_adsp/board.yml

```
@@ -0,0 +1,5 @@
+board:
+  name: amd_acp_6_0_adsp
```

please drop "amd_" prefix from the board name. Also needs a full name (see suggested change) 

```suggestion
  name: acp_6_0_adsp
  full_name: ACP_6_0
```

---

full_name still missing